man/systemd.special.xml

```diff
@@ -328,10 +328,11 @@
       <varlistentry>
         <term><filename>kbrequest.target</filename></term>
         <listitem>
-          <para>systemd starts this target whenever Alt+ArrowUp is
-          pressed on the console. This is a good candidate to be
-          aliased (symlinked) to
-          <filename>rescue.target</filename>.</para>
+          <para>systemd starts this target whenever Alt+ArrowUp is pressed on the
+          console. It may be useful to alias this unit to
+          <filename>debug-shell.service</filename> as a last ditch debugging
+          technique. (Note that this is for <emphasis>debugging</emphasis>, and
+          leaves the machine vulnerable to local attackers).</para>
```

Comment:
Indeed. I'll add the idea to https://freedesktop.org/wiki/Software/systemd/Debugging/. I changed the text to just give a generic warning without suggesting anything to alias to. Once we have a good emergency shell with authentication, we can revisit this.